tests/run/for_in_iter.py

```diff
@@ -75,21 +75,21 @@ def listcomp_over_multiplied_constant_tuple():
         [i for i in (1, 2, 3) * 2],
     ]
 
-
-@cython.test_assert_path_exists('//ReturnStatNode//ForInStatNode//TupleNode')
-@cython.test_fail_if_path_exists('//ReturnStatNode//ForInStatNode//ListNode')
-def listcomp_over_multiplied_constant_list():
-    """
-    >>> listcomp_over_multiplied_constant_list()
-    [[], [1, 2, 3], [1, 2, 3, 1, 2, 3], [1, 2, 3, 1, 2, 3, 1, 2, 3], [1, 2, 3, 1, 2, 3]]
-    """
-    return [
-        [i for i in [1, 2, 3] * 0],
-        [i for i in [1, 2, 3] * 1],
-        [i for i in [1, 2, 3] * 2],
-        [i for i in [1, 2, 3] * 3],
-        [i for i in [1, 2, 3] * 2],
-    ]
+# FIXME this is a nightmare to test with loops that are produced in IterationTransform
```

Comment:
But it's still a valid test, isn't it?

Reply:
It is a valid test. The problem goes away though if I don't apply this to tuples and lists (since optimized code for them exists elsewhere). It mainly useful to test it on tuples/lists while developing as a bug-finding exercise.